src/sentry/static/sentry/app/views/organizationGroupDetails/header.tsx

```diff
@@ -291,3 +298,14 @@ const EventAnnotationWithSpace = styled(EventAnnotation)`
 export {GroupHeader, TAB};
 
 export default withApi(GroupHeader);
+
+const StyledListLink = styled(ListLink)`
+  ${p =>
+    !p.to &&
+    `
+    &.active a, &.active a:focus &.active a:hover, a:hover, a.active {
+      color: #7c6a8e !important;
```

Comment:
Should this color come from theme?

Reply:
I just copied the same color that we are using in the less file and unfortunately, it is not part of our theme